src/components/ResultsMap.vue

```diff
@@ -44,9 +50,9 @@ export default {
 				};
 			}
 		},
-		handleMarkerClick(name) {
-			this.centerMapToClickedLocation(name);
-			this.setSelectedMarker(name);
+		handleMarkerClick(event, marker) {
```

Comment:
Do we need to pass `event` here if it isn't being used within `handleMarkerClick`?

Reply:
Just removed it, thanks for pointing this out!